test/signalrclienttests/CMakeLists.txt

```diff
@@ -24,5 +24,6 @@ include_directories(
 
 add_executable (signalrclienttests ${SOURCES})
 
-target_link_libraries(signalrclienttests gtest signalrclient ${CPPREST_LIB})
+find_package(Boost REQUIRED COMPONENTS system)
+target_link_libraries(signalrclienttests gtest signalrclient ${CPPREST_LIB} Boost::system)
```

Comment:
Just remove CPPREST_LIB from being linked, tests no longer need it

Reply:
But you'll got an linking error when you are compile with cpprestsdk download from apt(even if the `CMakeLists.txt` doesn't specify `${CPPREST_LIB}`)

```
/usr/bin/ld: CMakeFiles/signalrclienttests.dir/connection_impl_tests.cpp.o: undefined reference to symbol '_ZN5boost6system15system_categoryEv'
```

There is an [issue](https://github.com/microsoft/cpprestsdk/issues/792) in cpprestsdk's repo. It might help.